developer_manual/app/javascript-apis.rst

```diff
@@ -0,0 +1,157 @@
+`Read the full documentation <https://nextcloud.github.io/nextcloud-axios/>`_
+
+
+``@nextcloud/event-bus``
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+This package provides a simple event bus implementation that integrates with server and other apps. Thus it is one of the recommended ways of inter-app communication.
+`Read the full documentation <https://nextcloud.github.io/nextcloud-event-bus/>`_
+
+
+``@nextcloud/dialogs``
+^^^^^^^^^^^^^^^^^^^^^^
+
+This package provides access to UI dialogs in Nextcloud.
+`Read the full documentation <https://nextcloud.github.io/nextcloud-dialogs/>`_
+
+
+``@nextcloud/files``
+^^^^^^^^^^^^^^^^^^^^
+
+This package provides helper functions around the Files app.
+`Read the full documentation <https://nextcloud.github.io/nextcloud-files/>`_
+
+
+``@nextcloud/initial-state``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This package provides the counterpart for the `\OCP\IInitialStateService` on the back-end. Use it to retreive your stored data on page load.
+`Read the full documentation <https://nextcloud.github.io/nextcloud-initial-state/>`_
+
+
+``@nextcloud/l10n``
+^^^^^^^^^^^^^^^^^^^
+
+This package provides everything around localization, like access to the current user's locale and translation helpers.
+`Read the full documentation <https://nextcloud.github.io/nextcloud-l10n/>`_
+
+
+``@nextcloud/logger``
+^^^^^^^^^^^^^^^^^^^^^
+
+This package provides a unified logging helper that adds app names, severity and other context to log messages. Use it to enhance the output of your app's logs, useful for development and triaging bug reports.
+`Read the full documentation <https://nextcloud.github.io/nextcloud-logger/>`_
+
+
+``@nextcloud/moment``
+^^^^^^^^^^^^^^^^^^^^^
+
+This package provides a modified version of `moment.js <https://momentjs.com/>`_ with the current user's locale set.
+`Read the full documentation <https://nextcloud.github.io/nextcloud-moment/>`_
+
+
+``@nextcloud/password-confirmation``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This package makes it possible to ask a user for confirmation on actions that have a `@PasswordConfirmationRequired` set on the controller method. Use it for critical actions.
+`Read the full documentation <https://nextcloud.github.io/nextcloud-password-confirmation/>`_
+
+
+``@nextcloud/paths``
+^^^^^^^^^^^^^^^^^^^^
+
+This package provides various helpers for file and folder paths.
+`Read the full documentation <https://nextcloud.github.io/nextcloud-paths/>`_
+
+
+``@nextcloud/router``
+^^^^^^^^^^^^^^^^^^^^^
+
+This package provides helpers to generate URLs, e.g. to access assets and REST APIs of your app or the Nextcloud server.
+`Read the full documentation <https://nextcloud.github.io/nextcloud-router/>`_
+
+
+``@nextcloud/vue``
+^^^^^^^^^^^^^^^^^^
+
+This package provides a component library.
+`Read the full documentation <https://github.com/nextcloud/nextcloud-vue/#documentation>`_
```

Comment:
I'd say we can link directly to https://nextcloud-vue-components.netlify.com/

Reply:
sure! I wasn't sure if this was the final domain. but we of course always update the docs if the domain changes :)